### ISSUE TITLE
Schedules disabled by default?

### ISSUE THREAD

**Issue by aniceberg** [nielsfaber/scheduler-card#379]
Every time Home Assistant is rebooted, I see that the schedule toggle switched are turned off (or at least appear that way, even if the schedule is actually active?)

Is there an option to have the schedules ENABLED by default after reboot?

**Comment by nielsfaber** [comment]
Schedules should keep their state after rebooting.
They should neither disable or enable due to restarting. 
If you see this happening, you probably found a bug, and I would like to know more about it 🙂

**Comment by aniceberg** [comment]
On second thought, I don't it's related to the reboot process, it's only that I've noticed the change each time after I've applied an update to Hassio Core or Supervisor and rebooted it.

FWIW, just now I checked and noticed that the 1 daily schedule I have created was disabled, so I enabled it, rebooted the host, and it was still enabled after it came back up. The other 2 schedules I have created are only for Friday and Saturday respectively and both were enabled before and after rebooting.

If I notice the daily schedule disabled again tomorrow, and the weekend schedules disabled after the weekend, then I think it would be worthwhile to reopen this issue.
